### ISSUE TITLE
Feature request: Custom Minecraft installation path and improved detection

### ISSUE THREAD

**Issue by AgainPsychoX** [minity-script/minity#6]
Cannot use `minity links` command if Minecraft is installed in non-custom path, like via MultiMC launcher. 

```
PS C:\Users\PsychoX\Desktop\minecarts-nowadays> minity links
Error: ENOENT: no such file or directory, scandir 'C:\Users\PsychoX\AppData\Roaming\.minecraft\saves'
Exiting.
```

Some common options like MultiMC installations could be managed automatically (by looking in registry for MultiMC paths), then user could point which exact installation could be affected.

Anyway, minimal feature required would include some parameter to specify Minecraft saves folder by parameter (and/or maybe environment variable).

**Comment by soshimee** [comment]
Exactly! I use MultiMC as well because I have to manage multiple versions, but I noticed that this doesn't work. At least it should let us specify custom saves directory.
